driver/flags_table.c

```diff
@@ -111,6 +111,13 @@ const struct ppm_name_value clone_flags[] = {
 	{"NAME_CHANGED", PPM_CL_NAME_CHANGED},
 	{"CLOSED", PPM_CL_CLOSED},
 	{"CLONE_NEWUSER", PPM_CL_CLONE_NEWUSER},
+	{"CLONE_CHILD_CLEARTID", PPM_CL_CLONE_CHILD_CLEARTID},
+	{"CLONE_CHILD_SETTID", PPM_CL_CLONE_CHILD_SETTID},
+	{"SIGCHLD", PPM_CL_SIGCHLD},
```

Comment:
We have already a `PT_SIGTYPE` type that represents a signal. You can add a new argument on `clone` that reports this signal and it should be represented correctly by sysdig. I assume there will be 0 if nothing else has been set from user.